holidayextras/brands
# UXUI-539, Adding a SAGA Theme

#### What does this PR do? (please provide any background)
We want a boiler plate SAGA theme base, this showcases the work myself and @stuartgiles did on demo day. This is by no means a finished brand but certainly a good place to start. I will create a github issue to complete further work on this.

#### What tests does this PR have?
None

#### How can this be tested?
Checkout the branch in pattern library and feel free to add notes to the github issue on what needs adding. The benefit of the PL is that we can instantly see what needs adding in the brand.

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
